airflow/example_dags/example_datasets.py

```diff
@@ -70,21 +69,17 @@
 with DAG(
```

Comment:
Just wondering, is there a specific reason why we’re a bit inconsistent in using the context manager or not in this file?

Reply:
I'm not sure. We are inconsistent from example to example but not within example files. I can unify them here and update the dedent in the docs too.